aptos-move/framework/src/built_package.rs

```diff
@@ -93,7 +93,7 @@ pub(crate) fn build_model(
         test_mode: false,
         force_recompilation: false,
         fetch_deps_only: false,
-        fetch_latest_git_deps: false,
+        skip_fetch_latest_git_deps: false,
```

Comment:
@gregnazario , @wrwg , I had to change the name from `fetch_latest_git_deps` to `skip_fetch_latest_git_deps` to reflect https://github.com/move-language/move/pull/588. I'd like to make sure the initial value here (`false`) is still correct?

Reply:
Recommend to set this to true without guidance from @gregnazario so we do not break anything here. @davidiw any different take?

Reply:
OK. I set it to be true.